Test2/Tests/SettingPageBase.cs

```diff
@@ -0,0 +1,150 @@
+             .Until(ExpectedConditions.AlertIsPresent());
+        }
+
+        [Test]
+
+        public void CurrencyVer()
+        {
+            List<string> actuale = new List<string>();
+            List<string> expected = new List<string>() { "$ - US dollar", "€ - euro", "£ - Great Britain Pound", "₴ - Ukrainian hryvnia" };
+            IWebElement curren = driver.FindElement(By.Id("currency"));
+            SelectElement currenSeletct = new SelectElement(curren);
+            foreach (IWebElement element in currenSeletct.Options)
+            {
+                actuale.Add(element.Text);
+            }
+            Assert.AreEqual(expected, actuale);
+        }
+
+        [TestCase("dd/MM/yyyy")]
+        [TestCase("dd-MM-yyyy")]
+        [TestCase("MM/dd/yyyy")]
+        [TestCase("MM dd yyyy")]
+
+        public void DateFormat(string format)
+        {
+            IWebElement dateForm = driver.FindElement(By.XPath("//select[@id='dateFormat']"));
+            SelectElement dateFormSeletct = new SelectElement(dateForm);
+            dateFormSeletct.SelectByText(format);
+            IWebElement btnSave = driver.FindElement(By.Id("save"));
+            btnSave.Click();
+            WaitForAlert();
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+             .Until(ExpectedConditions.TitleContains("Deposite"));
+            IWebElement endDate = driver.FindElement(By.Id("endDate"));
+            string expectedDate = DateTime.Now.ToString(format);
+            Assert.AreEqual(expectedDate, endDate.GetAttribute("value"));
+        }
+
+        [TestCase("$ - US dollar", "$")]
+        [TestCase("€ - euro", "€")]
+        [TestCase("£ - Great Britain Pound", "£")]
+        [TestCase("₴ - Ukrainian hryvnia", "₴")]
+
+        public void Currency(string curren,string simcur)
+        {
+            IWebElement currenc = driver.FindElement(By.XPath(".//select[@id='currency']"));
+            IWebElement btnSave = driver.FindElement(By.Id("save"));
+            SelectElement currencSelect = new SelectElement(currenc);
+            currencSelect.SelectByText(curren);
+            WaitForReady();
+            btnSave.Click();
+            WaitForAlert();
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.UrlContains("Calculator"));
+            string ActualUrl = driver.Url;
+            string expectedUrl = "https://localhost:5001/Calculator";
+            Assert.AreEqual(expectedUrl, ActualUrl);
+            IWebElement simb = driver.FindElement(By.Id("currency"));
+            Assert.AreEqual(simcur, simb.Text);
+        }
+
+        [Test]
+
+        public void Logut()
+        {
+            IWebElement logut = driver.FindElement(By.XPath("//div[@class='login link btn btn-link']"));
```

Comment:
You can use `contains` for long classes:
```suggestion
            IWebElement logut = driver.FindElement(By.XPath("//div[contains(@class, 'login') ]"));
```